.docker/mysql/schema/list_update_bigint_ids.sql

```diff
@@ -0,0 +1,59 @@
+CREATE DATABASE IF NOT EXISTS `readitla_ril-tmp`;
+
+USE `readitla_ril-tmp`;
+--  The same schema as list, except uses expanded types for item_id and resolved_id.
+create table list_schema_update
+(
+    user_id        int unsigned               not null,
+    item_id        bigint unsigned            not null,
+    resolved_id    bigint unsigned            not null,
+    given_url      text                       not null,
+    title          varchar(75)                not null,
+    time_added     datetime                   not null,
+    time_updated   datetime                   not null,
+    time_read      datetime                   not null,
+    time_favorited datetime                   not null,
+    api_id         mediumint unsigned         not null,
+    status         tinyint unsigned           not null,
+    favorite       tinyint unsigned default 0 not null,
+    api_id_updated mediumint unsigned         not null,
+    primary key (user_id, item_id)
+)
+    collate = utf8_unicode_ci;
+
+--  Also create indices, so as to keep them updated while backfilling.
```

Comment:
Is this cause not have the same index names?

Reply:
I assume so -- this I just copied from Marion's web repo PR